app/models/team.rb

```diff
@@ -564,8 +564,10 @@ def filter_by_keywords(query, filters, type = 'FactCheck')
   end
 
   def search_for_similar_articles(query, pm = nil)
-    # query expected to be text
-    result_ids = Bot::Smooch.search_for_similar_published_fact_checks_no_cache('text', query, [self.id]).map(&:id)
+    # query:  expected to be text
+    # pm: to request a most relevant to specific item and also include both FactCheck & Explainer
+    limit = pm.nil? ? CheckConfig.get('most_relevant_team_limit', 3, :integer) : CheckConfig.get('most_relevant_item_limit', 10, :integer)
+    result_ids = Bot::Smooch.search_for_similar_published_fact_checks_no_cache('text', query, [self.id], limit).map(&:id)
```

Comment:
If we are calling via the `_published_fact_checks` endpoint, how do we support the requirement to return also the unpublished fact_checks for the 'relevant' use case?

Reply:
There is a filter for published report so we can skip it to return both published & unpublished fact-checks
we can move the conversation to jira ticket as I'll merge the PR to epic branch